web/src/message_lists.ts

```diff
@@ -51,16 +52,18 @@ export function set_home(msg_list: MessageList): void {
 
 export function all_rendered_message_lists(): MessageList[] {
     assert(home !== undefined);
-    assert(current !== undefined);
     const rendered_message_lists = [home];
-    if (current !== home && !recent_view_util.is_visible()) {
+    if (current !== undefined && current !== home && !recent_view_util.is_visible()) {
```

Comment:
We get to drop the `recent_view_util.is_visible` check here, right? And I guess it was a bug that this wasn't checking inbox too, oops.

Reply:
oh yeah! Updated.